<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2121</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gestion Pedidos</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_proveedores">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Nombre">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Introduce el nombre del repuesto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Cantidad">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Introduce la cantidad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRealizarPedido">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>650</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Realizar pedido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>650</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnadirAlPedido">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>540</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AÃ±adir al pedido</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaRepuestosPedidos">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>391</width>
      <height>192</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="btnEliminarDelPedido">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>540</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar del pedido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PrecioUnitario">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Introduce el precio por unidad</string>
    </property>
   </widget>
   <widget class="QWidget" name="imagen" native="true">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>50</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
